src/components/Answers.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+function Answers(props) {
+  const { answers, shouldShowCorrectAnswers } = props;
+
+  const getCorrectOrWrongAnswerList = answers.filter((answer) => (
+    shouldShowCorrectAnswers === answer.correct));
+  return (
+    <div>
+      <h2>
+        {shouldShowCorrectAnswers ? 'Correct ' : 'Wrong '}
+        Answers Are
+      </h2>
+      <table className="Table">
+        <thead>
+          <tr>
+            <th className="TableHeaderRow">No</th>
+            <th className="TableHeaderRow">Question</th>
+            <th className="TableHeaderRow">Response</th>
+            <th className="TableHeaderRow">Answer</th>
+          </tr>
+        </thead>
+        <tbody>
+          {getCorrectOrWrongAnswerList.map((filteredAnswer) => (
```

Comment:
what does filteredAnswer mean? Does the var name make sense in the context of the map?

Reply:
Also use object destructuring.

Reply:
Done